app/controllers/sessions_controller.rb

```diff
@@ -5,11 +5,20 @@ def create
     user = User.find_by(email: params[:session][:email].downcase)
      
     if user&.authenticate(params[:session][:password])
-      reset_session
-      log_in user
-      params[:session][:remember_me] == '1' ? remember(user) : forget(user)
 
-      redirect_to user
+      if user.activated?
+        reset_session
+        log_in(user)
+        params[:session][:remember_me] == '1' ? remember(user) : forget(user)
+
+        redirect_to(user)
+      else 
+        message = 'Account not activated. '
+        message += 'Check your email for the activation link.'
```

Comment:
You should set the message in one single line, no need to concatenate on a second line